package.json

```diff
@@ -102,7 +102,9 @@
     "glob-parent": "^5.1.2",
     "path-parse": "^1.0.7",
     "ua-parser-js": "^0.7.22",
-    "underscore": "1.13.1"
+    "underscore": "1.13.1",
+    "ansi-regex": "5.0.1",
+    "normalize-url": "6.0.1"
```

Comment:
It seems we can use here `"patternfly-react/**/normalize-url": "4.5.1"` instead.

Details:
1. As per [release notes](https://github.com/sindresorhus/normalize-url/releases/tag/v6.0.1) for normalize-url versions below 4.3.0 are safe. The fix has been also backported to v4 branch as 4.5.1.
2. The only package that uses vulnerable version is patternfly-react (see output of `yarn why` below)
3. our CSS minimizer plugin uses v1 which is far away from v6 (with several breaking changes along the way) which creates regression risk on production.

 
```
» yarn why  normalize-url
//..
=> Found "normalize-url@1.9.1"
info Reasons this module exists
   - "mini-css-extract-plugin" depends on it
   - Hoisted from "mini-css-extract-plugin#normalize-url"
// ... 
=> Found "postcss-normalize-url#normalize-url@3.3.0"
info This module exists because "optimize-css-assets-webpack-plugin#cssnano#cssnano-preset-default#postcss-normalize-url" depends on it.
// ...
=> Found "@semantic-release/npm#normalize-url@4.5.0"
info This module exists because "patternfly-react#react-ellipsis-with-tooltip#semantic-release#@semantic-release#npm" depends on it.
```

Reply:
Done